main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "github.com/gin-gonic/gin"
+
+func main() {
+	router := gin.Default()
+
+	router.GET("/ping", func(context *gin.Context) {
+		context.JSON(200, gin.H{"message": "success"})
+		return
+	})
```

Comment:
this handler doesn't handle telegram's post request fix it

Reply:
ok